src/OmniSharp.Roslyn.CSharp/Services/Buffer/UpdateBufferService.cs

```diff
@@ -20,7 +20,7 @@ public UpdateBufferService(OmniSharpWorkspace workspace)
 
         public async Task<object> Handle(UpdateBufferRequest request)
         {
-            await _workspace.BufferManager.UpdateBuffer(request);
+            await _workspace.BufferManager.UpdateBufferAsync(request);
```

Comment:
http://stackoverflow.com/questions/15951774/does-the-use-of-the-async-suffix-in-a-method-name-depend-on-whether-the-async